test/testutil/driver.c

```diff
@@ -150,10 +150,28 @@ void set_test_title(const char *title)
     test_title = title == NULL ? NULL : strdup(title);
 }
 
+void test_verdict(int pass, const char *extra, ...)
```

Comment:
I'm using it in `test_main.c` to indicate an error in registration.
I could change that to a TEST_error or equivalent and make this static.